.gitlab-ci.yml

```diff
@@ -342,7 +342,7 @@ warnings:
     <<: *default_variables
     BUILD_OMP: "ON"
     BUILD_CUDA: "ON"
-    CXX_FLAGS: "-Werror=pedantic"
+    CXX_FLAGS: "-Werror=pedantic -pedantic-error"
```

Comment:
Should be [`-pedantic-errors`](https://gitlab.com/ginkgo-project/ginkgo-public-ci/-/jobs/171923275), I think

Reply:
My bad, you are definitely right.